### ISSUE TITLE
How to purchase license

### ISSUE THREAD

**Issue by davbaron** [blasten/turn.js#474]
We are going to use Turn.js in a commercial product. I thought I had seen a 'purchase' link somewhere on the main site, but cannot find it any longer.  I've written to Emmanuel, but have not heard back. Can anyone tell me where to purchase this from?

**Comment by filamananda** [comment]
This project is dead for what it seems. My bitter experience says, if you wish to use this for your production site, use another plugin; may be 'page-flip' or write one yourself.

**Comment by RPSource** [comment]
The 4th release files are currently downloadable through the website (just press the download button)

**Comment by davbaron** [comment]
Yes, the files are downloadable, however within the source code is a licensing statement that states you have to purchase the software.  I contacted Emmanuel about this, but he is non-responsive.  So... it appears we have 'free to use' software package, that is not really legally free to use (commercially).

**Comment by act2281** [comment]
http://www.turnjs.com/get

**Comment by jasonlav** [comment]
The get page (http://www.turnjs.com/get) isn't currently available from the homepage or licensing page, so I am concerned that it is out of date or no longer valid. Please advise.
